src/binary/Yast.cc

```diff
@@ -166,6 +168,81 @@ void set_ruby_source_location(VALUE file, VALUE lineno)
   YaST::ee.setLinenumber(FIX2INT(lineno));
 }
 
+/**
+ * Is the function name an UI function which returns a symbol?
```

Comment:
does it really check all methods that return symbol? I think it is only events that poll events and return it as symbol, not?